### PR TITLE
MAT-6729: Adding Cors attribute to allow the service to be called fro…

### PR DESCRIPTION
**Pull request by gregory-akins**
…m our front end

## MADiE PR

Jira Ticket: [MAT-6729](https://jira.cms.gov/browse/MAT-6729)
(Optional) Related Tickets:

### All Submissions

- [X] This PR has the JIRA linked.
- [X] Required tests are included.
- [X] No extemporaneous files are included (i.e Complied files or testing results).
- [X] This PR is merging into the **correct branch**.
- [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
- [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps

If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

- [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
- [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers

By Approving this PR you are attesting to the following:

- Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
- The tests appropriately test the new code, including edge cases.
- If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { NestFactory } from '@nestjs/core';
 import { ExportModule } from './export.module';
 
 export async function bootstrap() {
-  const app = await NestFactory.create(ExportModule);
+  const app = await NestFactory.create(ExportModule, { cors: true });
```
Comment (RohitKandimalla) [inline src/main.ts]: Why do we want to enable CORS ? Shouldn't our domain remain same for UI and backend servers

Comment (gregory-akins) [reply src/main.ts:5]: We have CORS for all services; port matters also

Comment (gregory-akins) [reply src/main.ts]: I thought we had CORS for all our services (Maybe I'm not remembering correctly).

**Review by RohitKandimalla — APPROVED** [review]
